src/commcare_cloud/commands/ansible/service.py

```diff
@@ -245,7 +261,7 @@ def execute_action(self, action, host_pattern=None, process_pattern=None):
 
         def _change_state(service_name, run_on, action=action):
             service_args = 'name={} state={}'.format(service_name, STATES[action])
-            return self._run_ansible_module(run_on, 'service', service_args)
+            return self._run_ansible_module(run_on, _ansible_module_helper(service_name), service_args)
```

Comment:
i see this is kind of clever. Basically this works because the `monit` ansible module and the `service` ansible module have similar interfaces. It's unlikely to cause us problems, but I might have preferred minor code duplication to treating service and monit modules as an interchangeable interface, which only happens to work here. Something like this

```
    if service_name in MONIT_MANAGED_SERVICES:
            service_args = 'name={} state={}'.format(service_name, STATES[action])
            return self._run_ansible_module(run_on, 'monit', service_args)
    else:
            service_args = 'name={} state={}'.format(service_name, STATES[action])
            return self._run_ansible_module(run_on, 'service', service_args)
```
in the helper function would better reflect that these are two different module calls with potentially different interfaces. (For example if we need to pass a special argument to monit to change the behavior in some way, we can then just edit that one.) Anyway, not a big deal/necessarily actionable, just trying to articulate what felt funny to me about this pattern.

Reply:
Yeah. I'm not married to the approach I took. I was hoping to make the `monit` services a subclass of `AnsibleService` but the multi service configurations made that a bit more difficult because couch has one service monitored by monit and another by service. Changing service configurations to fix that seemed like a larger change that wasn't worth it which is why I went the way I did.

Your approach seems fine as well. I haven't had to maintain/change this code before so I'd default to your judgement on the way to write it. Feel free to change it as you see fit